packages/kit/test/apps/basics/src/routes/ws/handle/+server.js

```diff
@@ -0,0 +1,6 @@
+export const socket = {
+	upgrade() {
+		// always abort the upgrade request because we just want to test the handle hook runs
+		throw new Response();
```

Comment:
is throwing a response idiomatic?

Reply:
Agreed we'll deviate from crossws here and disallow throwing or returning Response-like objects in the `upgrade` hook.